src/Networking/IPAddress.cs

```diff
@@ -7,7 +7,7 @@ namespace CSM.Networking
     public class IPAddress
     {
         private static string _localIp = null;
-        private static string _externalIp = null;
+        public static string _externalIp = null;
```

Comment:
The variable should be private, only accessible with the method below, i think.

Reply:
Since the server list would send the given external ip as SQL to a database it was better to change the existing one ready for another commit later on. Though it could always return to its previous one for now.